deployment/deployutils/wizardInputs.cpp

```diff
@@ -316,6 +316,9 @@ CInstDetails* CWizardInputs::getServerIPMap(const char* compName, const char* bu
  
       if(m_compIpMap.find(buildSetName) != NULL)
       {
+        if(instDetails != NULL)
+          delete(instDetails);
```

Comment:
@rpastrana 
This is being picky, and would have ignored if only 1 of them, but a couple of issues
i) no need to check for null before calling delete.
ii) this looks like a function call rather than a use of the delete operator.

`delete instDetails;`

is much more common.